libs/models/player.ts

```diff
@@ -0,0 +1,19 @@
+interface TimeLog {
+  length: number; //typed text length.
+  distance: number; //time distance between initial point and current timestamp.
```

Comment:
the `currentTimestamp` must also be there right?

Reply:
I think we don't need to send the current timestamp every time as we send the timestamp when the player starts. And after that, we send elapsed time in seconds (local) for every 2 seconds (only an example) to the server. So when the server receives the related message, it can calculate the latency and other stuff. It's almost like sending unix timestamp but we only send the difference between the current and started timestamp.

Reply:
Discussed and decided to put `timestamp`